library/core/src/slice/mod.rs

```diff
@@ -1314,7 +1314,6 @@ impl<T> [T] {
     /// # Examples
```

Comment:
In the section above (which I cannot comment because it is too far from the diff) we need to strike _"This check will most probably get changed to a compile time error before this method gets stabilized."_